apidoc/Titanium/UI/TextField.yml

```diff
@@ -621,10 +621,20 @@ events:
 
   - name: return
     summary: Fired when the return key is pressed on the keyboard.
+    description: |
+            Since in Landscape mode Android provides both the New Line and the Action type
+            buttons as return keys the 'button' property in the event is used to distinguish from
+            which key has the event been fired.
```

Comment:
> Since Android provides both the new line- and the action buttons as return keys in landscape mode, the `button` property in the event is used to distinguish from which key the event has been fired.

In general: I think @jquick-axway can give some language advices here as well.